src/ppc_cpuid.c

```diff
@@ -28,6 +28,8 @@
 // if the user runs this on a machine which the SIMD is there, but FAILS to be new enough
 // then they will have to add the --disable-simd flag to configure.
 int main(int argc, char **argv) {
+	if (argc==1)
+		return !printf("This program a Linux system to run with any meaningful data\n");
```

Comment:
This sentence is not quite well formed.

Reply:
> I made sure it was exactly like what is in bleeding-jumbo, to avoid merge conflict issues

Your workarounds are strange and confusing: Why didn't you just rebase the code upon current tree before pushing it here?! There's normally no manual job in doing so at all.

Reply:
I hurried to merge (not as fast as I should, but anyone could do it "for me", if needed). 
* In case of doubt, just ask if I was able to proper test (in this case, yes, but only after the PR).

Reply:
@jfoug if I understand you right, you are completely misunderstanding how Git works. If you don't "backport" commits things like this from branches that are ahead of your topic branch, Git is very likely to sort things out, because it knows at what point in time your branch forked from bleeding-jumbo. Hell, I can even merge core stuff into bleeding-jumbo without too much work. But when you **do** add out-of-branch commits, it will almost certainly **make** Git fail.